fc-java-events/src/main/java/com/aliyun/fc/runtime/event/OSSEvent.java

```diff
@@ -1,133 +1,250 @@
 package com.aliyun.fc.runtime.event;
 
-import com.fasterxml.jackson.annotation.JsonCreator;
-import com.fasterxml.jackson.annotation.JsonProperty;
+public class OSSEvent {
 
-public final class OSSEvent {
-
-    public final Event events[];
-
-    @JsonCreator
-    public OSSEvent(@JsonProperty("events") Event[] events) {
-        this.events = events;
-    }
+    public OSSEvent.Event events[];
 
     public static final class Event {
 
-        public final String eventName;
-        public final String eventSource;
-        public final String eventTime;
-        public final String eventVersion;
-        public final Oss oss;
-        public final String region;
-        public final RequestParameters requestParameters;
-        public final ResponseElements responseElements;
-        public final UserIdentity userIdentity;
-
-        @JsonCreator
-        public Event(@JsonProperty("eventName") String eventName,
-            @JsonProperty("eventSource") String eventSource,
-            @JsonProperty("eventTime") String eventTime,
-            @JsonProperty("eventVersion") String eventVersion,
-            @JsonProperty("oss") Oss oss,
-            @JsonProperty("region") String region,
-            @JsonProperty("requestParameters") RequestParameters requestParameters,
-            @JsonProperty("responseElements") ResponseElements responseElements,
-            @JsonProperty("userIdentity") UserIdentity userIdentity) {
+        private String eventName;
+        private String eventSource;
+        private String eventTime;
+        private String eventVersion;
+        private OSSEvent.Event.Oss oss;
+        private String region;
+        private OSSEvent.Event.RequestParameters requestParameters;
```

Comment:
I prefer a flat structure instead of nest one. e.g. all other fields are declared under OSSEvent.

So user write `OSSEvent.Bucket` instead of `OSSEvent.Event.Oss.Bucket`